samples/deployment/test-network/README.md

```diff
@@ -26,68 +27,52 @@ Use `CC_ID` and `CC_PATH` to define the FPC Chaincode you want to build.
 cd $FPC_PATH/samples/deployment/test-network
 export CC_ID=echo
 export CC_PATH=$FPC_PATH/samples/chaincode/echo
+# TODO this needs to be changed!
+export CC_VER=$(cat CC_PATH/)
```

Comment:
this doesn't work

Reply:
Didn't work with me either. I also tried to type `export CC_VER=$(echo $CC_PATH)` as cat was erroring out `No such file or directory` but the `make` also failed with 
```
# github.com/hyperledger/fabric-private-chaincode/ercc/attestation
attestation/pdo.go:16:19: undefined: pdo.NewEpidLinkableVerifier
attestation/pdo.go:17:19: undefined: pdo.NewEpidUnlinkableVerifier
make[1]: *** [Makefile:18: ercc] Error 1
```

Reply:
I think it should point to the `mrenclave` file like the example here https://github.com/munapower/fabric-private-chaincode/tree/upgradeTestNetwork/samples/chaincode/simple-asset-go#start-the-test-network

Reply:
After applying 
```
cd $FPC_PATH/samples/deployment/test-network
export CC_ID=echo
export CC_PATH=$FPC_PATH/samples/chaincode/echo
export CC_VER=$(cat $CC_PATH/mrenclave)
make build
```
It failed with 
```
docker  build --build-arg FPC_VERSION=main -t fpc/fpc-echo:main -f Dockerfile.fpc-app \
                --build-arg HW_EXTENSION=\
 		
ERROR: "docker buildx build" requires exactly 1 argument.
See 'docker buildx build --help'.

Usage:  docker buildx build [OPTIONS] PATH | URL | -

Start a build
make[1]: *** [Makefile:73: docker-fpc-app] Error 1
make[1]: Leaving directory '/project/src/github.com/hyperledger/fabric-private-chaincode/ecc'
make: *** [Makefile:22: ecc-container] Error 2
```

Reply:
The `Makefile` in the `$FPC_PATH/samples/deployment/test-network` is not working for any example I test. But the `Makefile` inside `echo-go` and `simple-asset-go` works. 
It's noteworthy that even the `fpc-echo-go` image was built successfully and the `./installFPC` worked for it but I wasn't able to initialize the enclave and faced this error
```
cd $FPC_PATH/samples/application/simple-go
CC_ID=echo ORG_NAME=Org1 go run . -withLifecycleInitEnclave
2024-07-23 21:52:51.870 UTC 0001 INFO [sdk-test] main -> --> Invoke LifecycleInitEnclave
2024-07-23 21:52:51.885 UTC 0002 FATA [sdk-test] main -> Failed to invoke LifecycleInitEnclave: Failed to query init enclave: Transaction processing for endorser [localhost:7051]: Endorser Client Status Code: (23) CHAINCODE_NAME_NOT_FOUND. Description: make sure the chaincode echo has been successfully defined on channel mychannel and try again: chaincode echo not found
exit status 1
```